src/index.ts

```diff
@@ -332,58 +341,193 @@ async function invokeLambda(payload: OperationsInput): Promise<OperationsOutput>
+     */
     async getInstance(input: GetInstance): Promise<CloudObjectResponse | undefined> {
         return this.sendSingleOperation(input, this.getInstance.name)
     }
+
+    /**
+     *
+     * Gets the state of an instance
+     * @param {GetInstance} input
+     * @return {*}  {(Promise<CloudObjectResponse | undefined>)}
+     * @memberof CloudObjectsOperator
+     */
     async getState(input: GetInstance): Promise<CloudObjectResponse | undefined> {
         return this.sendSingleOperation(input, this.getState.name)
     }
+
+    /**
+     *
+     * Gets the instance id coressponds to the given lookup key
+     * @param {LookUpKey} input
+     * @return {*}  {(Promise<OperationResponse | undefined>)}
+     * @memberof CloudObjectsOperator
+     */
     async getLookUpKey(input: LookUpKey): Promise<OperationResponse | undefined> {
         return this.sendSingleOperation(input, this.getLookUpKey.name)
     }
+
+    /**
+     *
+     * Sets the current instance id to the given lookup key
+     * @param {LookUpKey} input
+     * @return {*}  {(Promise<OperationResponse | undefined>)}
+     * @memberof CloudObjectsOperator
+     */
     async setLookUpKey(input: LookUpKey): Promise<OperationResponse | undefined> {
         return this.sendSingleOperation(input, this.setLookUpKey.name)
     }
+
+    /**
+     *
+     * Deletes the given lookup key
+     * @param {LookUpKey} input
+     * @return {*}  {(Promise<OperationResponse | undefined>)}
+     * @memberof CloudObjectsOperator
+     */
     async deleteLookUpKey(input: LookUpKey): Promise<OperationResponse | undefined> {
         return this.sendSingleOperation(input, this.deleteLookUpKey.name)
     }
+
+    /**
+     *
+     * Sets the value to the given key in memory
+     * @param {SetMemory} input
+     * @return {*}  {(Promise<OperationResponse | undefined>)}
+     * @memberof CloudObjectsOperator
+     */
     async setMemory(input: SetMemory): Promise<OperationResponse | undefined> {
         return this.sendSingleOperation(input, this.setMemory.name)
     }
+
+    /**
+     *
+     * Gets the value of the given key from memory
+     * @param {GetMemory} input
+     * @return {*}  {(Promise<OperationResponse | undefined>)}
+     * @memberof CloudObjectsOperator
+     */
     async getMemory(input: GetMemory): Promise<OperationResponse | undefined> {
         return this.sendSingleOperation(input, this.getMemory.name)
     }
+
+    /**
+     *
+     * Deletes the given key from memory
+     * @param {DeleteMemory} input
+     * @return {*}  {(Promise<OperationResponse | undefined>)}
+     * @memberof CloudObjectsOperator
+     */
     async deleteMemory(input: DeleteMemory): Promise<OperationResponse | undefined> {
         return this.sendSingleOperation(input, this.deleteMemory.name)
     }
+
+    /**
+     *
+     * Increments the value of the given key in memory
+     * 
+     * The value have to be a number
+     * Input parameter path can be used to increment a nested key.
```

Comment:
"Input parameter path can be used to increment a nested key."
bu feature benım gozumden kacmıs ya. ellerine sağlık @cyberst 

adamın tacını getirin! 👑